### ISSUE TITLE
Lint: Field Numbers

### ISSUE THREAD

**Issue by emilrowland** [yoheimuta/protolint#453]
The linter doesn't detect if multiple fields share the same Field numbers.
Field numbers should always be unique.

Example should not be allowed:
```
message Example{
  string field1 = 1;
  string field2 = 1;
}
``
